### PR TITLE
deprecate `package update`

### PR DESCRIPTION
**Pull request by tamarrow**

### CODE REVIEW

File: cli/dcoscli/package/main.py [hunk 1]
```diff
@@ -160,9 +160,10 @@ def _update(validate):
     :rtype: int
     """
 
-    config = util.get_config()
+    emitter.publish("This command is being deprecated")
```
Comment (jsancio) [inline cli/dcoscli/package/main.py]: Hmm. I think we should only print this when the CLI is configure to talk to `Cosmos`. Eventually, we are going to add code that auto discover if Cosmos is installed and configure it automatically.

**Comment by jsancio** [comment]
LGTM. Just one comment to address. Let me know what you think. Thanks!

File: cli/dcoscli/package/main.py (hunk 1 above)
Comment (tamarrow) [inline cli/dcoscli/package/main.py]: sgtm

**Comment by tamarrow** [comment]
@jsancio updated

File: cli/tests/integrations/test_package.py [hunk 2]
```diff
@@ -62,10 +66,13 @@ def test_update_with_validation():
...
-    assert b'source' in stdout
-    assert b'Validating package definitions...' in stdout
-    assert b'OK' in stdout
     assert stderr == b''
+    if os.environ.get(constants.COSMOS_URL_ENV) is None:
+        assert b'source' in stdout
+        assert b'Validating package definitions...' in stdout
+        assert b'OK' in stdout
+    else:
+        assert stdout == b'This command is being deprecated\n'
```
Comment (jsancio) [inline cli/tests/integrations/test_package.py]: Sorry should have mentioned this before. "This command is deprecated\n". From their point of view it already happened. :)

**Comment by jsancio** [comment]
One minor wording comment. Looks good to me!

**Comment by jsancio** [comment]
:ship: it

**Pull request merged**